test/controllers/users_controller_test.rb

```diff
@@ -10,7 +10,7 @@ class UsersControllerTest < ActionDispatch::IntegrationTest
 
   describe 'create method' do
     it 'should raise if user is not admin' do
-      User::ROLE.reject { |role| role == :admin }.each do |role|
+      User.role.values.reject { |role| role == :admin }.each do |role|
```

Comment:
Yeah, I was really confused how the double colons worked in general. Enumerize just makes more sense.